### ISSUE TITLE
A11y-Issues with "command buttons" 

### ISSUE THREAD

**Issue by MarcusMorba** [vaadin/vaadin-upload#311]
### Description
Command buttons (start, retry, clear) are not accessible as they are DIVs without role nor tabindex.

### Expected outcome
<button> markup.

### Actual outcome
Not accessible.

### Live Demo
https://vaadin.com/components/vaadin-upload/html-examples

### Steps to reproduce
Try to TAB elements.

### Browsers Affected

- [x ] Chrome
- [x ] Firefox
- [x ] Safari
- [x ] Edge
- [x ] IE 11
- [x ] iOS Safari
- [x ] Android Chrome

**Comment by web-padawan** [comment]
Thanks for the issue. I will make sure this gets covered by the LitElement rewrite, once we get to it.

**Comment by anezthes** [comment]
Ideally replaced with icon-only vaadin-buttons that have aria-labels. The aria-labels should also IMO reference the associated file (or allow for the option to do so), e.g. "Retry uploading Financials.xlsx" instead of just "Retry".

**Comment by web-padawan** [comment]
Closing in favor of https://github.com/vaadin/web-components/issues/121 which describes the same problem. 
We are going to start working on a fix for this and other a11y issues during the upcoming weeks.
